package.json

```diff
@@ -6,6 +6,9 @@
   "contributors": [
     "Juan Carlos Farah"
   ],
+  "workspaces": [
+    "example"
```

Comment:
Why do we need this?

Reply:
It actually ease the process, as it `yarn` the workspaces at the same time as the core repository.